pkg/ext-proc/handlers/request.go

```diff
@@ -70,23 +71,23 @@ func (s *Server) HandleRequestBody(reqCtx *RequestContext, req *extProcPb.Proces
 		klog.V(logutil.VERBOSE).Infof("Updated body: %v", string(requestBody))
 	}
 
-	targetPod, err := s.scheduler.Schedule(llmReq)
+	targetEndpoint, err := s.scheduler.Schedule(llmReq)
```

Comment:
I think calling this `targetPod` is fine here, as the return value is an internal Pod object. If you change it, pls also change the name in reqCtx and scheduler to be consistent.

Reply:
sg